### PR TITLE
is_staff also true if in FT offices IP range

### PR DESCRIPTION
**Pull request by georgecrawford**

### CODE REVIEW

File: server/pipelines/v3.js [hunk 1]
```diff
@@ -150,6 +147,12 @@ class Pipeline extends EventEmitter {
 			// Anything that we are just testing
 			event.annotate('experimental', {});
 
+			const user = event.annotations().user;
```
Comment (georgecrawford) [inline server/pipelines/v3.js]: Is this OK? I wasn't sure how else to do it.

**Comment by georgecrawford** [comment]
See also maxmind-service/pull-requests/4

**Comment by georgecrawford** [comment]
Closes #120.

File: server/pipelines/v3.js [hunk 2]
```diff
@@ -150,6 +147,12 @@ class Pipeline extends EventEmitter {
...
+			event.annotate('user', Object.assign(user, {
```
Comment (commuterjoy) [inline server/pipelines/v3.js]: I suppose we could mask that object splicing behind the model - I might do that this afternoon in another PR.

Comment (georgecrawford) [inline server/pipelines/v3.js]: Thanks, sounds good.

**Pull request merged**